assets/js/hooks/open_emoji_popup.js

```diff
@@ -0,0 +1,61 @@
+/**
+const emojiButton = document.getElementById("emoji_button")
+
+if (emojiPopup != null) {
```

Comment:
Leaving commented-out code is considered bad practice, do you have any plans to implement this in the future or is it just another method of doing the emojis?

Reply:
that's old code, it will be removed, thanks for noticing